mapie/tests/test_classification.py

```diff
@@ -130,17 +214,25 @@
             random_state=42
         ),
         ParamsPredict(
-            include_last_label=True
+            include_last_label=True,
+            agg_scores="mean"
         )
     ),
 }
 
 COVERAGES = {
     "score": 8 / 9,
-    "score_cv": 1,
+    "score_cv_mean": 1,
+    "score_cv_crossval": 1,
     "cumulated_score_include": 1,
     "cumulated_score_not_include": 5/9,
     "cumulated_score_randomized": 1,
+    "cumulated_score_include_cv_mean": 1,
+    "cumulated_score_not_include_cv_mean": 1,
+    "cumulated_score_randomized_cv_mean": 1,
+    "cumulated_score_include_cv_crossval": 1,
+    "cumulated_score_not_include_cv_crossval": 1,
```

Comment:
There are many unit tests with coverage 1 and producing predictions full of True. Can we engineer one non-trivial example checking the most natural use of cross-conformal ?

Reply:
Yes you're right, I decreased the `alpha` value to 0.5. Some coverages are still equal to 1 but the predictions are no longer full of `True`.